Tutorials/Reduction/README.md

```diff
@@ -0,0 +1,25 @@
+# Reduction Case Study
+
+This repository hosts the sample code used in the HIP documentation.
```

Comment:
The Reduction example is used in the HIP documentation at ... (provide reference).

Reply:
Reference is provided.